bazel/dep/deps.bzl

```diff
@@ -39,19 +39,19 @@ def wasm_dependencies():
 
     _http_archive(
         name = "rules_proto",
-        sha256 = "602e7161d9195e50246177e7c55b2f39950a9cf7366f74ed5f22fd45750cd208",
-        strip_prefix = "rules_proto-97d8af4dc474595af3900dd85cb3a29ad28cc313",
+        sha256 = "9fc210a34f0f9e7cc31598d109b5d069ef44911a82f507d5a88716db171615a8",
+        strip_prefix = "rules_proto-f7a30f6f80006b591fa7c437fe5a951eb10bcbcf",
         urls = [
-            "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
-            "https://github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
+            "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/f7a30f6f80006b591fa7c437fe5a951eb10bcbcf.tar.gz",
+            "https://github.com/bazelbuild/rules_proto/archive/f7a30f6f80006b591fa7c437fe5a951eb10bcbcf.tar.gz",
         ],
     )
 
     _http_archive(
         name = "com_google_protobuf",
-        sha256 = "59621f4011a95df270748dcc0ec1cc51946473f0e140d4848a2f20c8719e43aa",
-        strip_prefix = "protobuf-655310ca192a6e3a050e0ca0b7084a2968072260",
-        url = "https://github.com/protocolbuffers/protobuf/archive/655310ca192a6e3a050e0ca0b7084a2968072260.tar.gz",
+        sha256 = "6dd0f6b20094910fbb7f1f7908688df01af2d4f6c5c21331b9f636048674aebf",
+        strip_prefix = "protobuf-3.14.0",
```

Comment:
We're updating protobuf from v3.9.1 to v3.14.0 (the same version as in Envoy), but that's only in Bazel builds to pull updated rules definitions. I'm leaving the prebuilt libraries for `make` builds at v3.9.1. Any objections?

Reply:
This sounds fine to me. We should get rid of the prebuilt protobuf lib. If people wants to use use protobuf, they should use bazel. I will work on this by reviving https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/pull/71.

Reply:
SGTM, thanks!